libraries/AP_Relay/AP_Relay.cpp

```diff
@@ -58,28 +58,28 @@ const AP_Param::GroupInfo AP_Relay::var_info[] = {
     // @DisplayName: First Relay Pin
     // @Description: Digital pin number for first relay control. This is the pin used for camera shutter control. Some common values are given, but see the Wiki's "GPIOs" page for how to determine the pin number for a given autopilot.
     // @User: Standard
-    // @Values: -1:Disabled,49:BB Blue GP0 pin 4,50:AUXOUT1,51:AUXOUT2,52:AUXOUT3,53:AUXOUT4,54:AUXOUT5,55:AUXOUT6,57:BB Blue GP0 pin 3,113:BB Blue GP0 pin 6,116:BB Blue GP0 pin 5,27:BBBMini Pin P8.17,101:MainOut1,102:MainOut2,103:MainOut3,104:MainOut4,105:MainOut5,106:MainOut6,107:MainOut7,108:MainOut8
+    // @Values: -1:Disabled,1:ZealotRelay1,2:ZealotRelay2,3:ZealotRelay3,4:ZealotRelay4,49:BB Blue GP0 pin 4,50:AUXOUT1,51:AUXOUT2,52:AUXOUT3,53:AUXOUT4,54:AUXOUT5,55:AUXOUT6,57:BB Blue GP0 pin 3,113:BB Blue GP0 pin 6,116:BB Blue GP0 pin 5,27:BBBMini Pin P8.17,101:MainOut1,102:MainOut2,103:MainOut3,104:MainOut4,105:MainOut5,106:MainOut6,107:MainOut7,108:MainOut8
```

Comment:
instead of adding more confusing values, since they vary widely and are inconsistent from autopilot to autopilot, be changed as suggested below to only include the most common (as the description describes these values)...the description already directs to the gpios page for reading the hwdef just like all other pin param (BTN,RPM,etc.)....these values are almost universally LED controls and there are many other pin defs for general purpose GPIOs besides these for various controllers..directing the user to the appropriate hwdef will minimize confusion I think..
```suggestion
    // @Values: -1:Disabled,50-63:Commonly the AUXOUT/FMU servo/motor outputs,101-108:IOMCU Main servo/motor outputs
```

Reply:
PS going forward we might want to be more consistent in these defs when hwdefs are reviewed...

Reply:
@Hwurzburg, txs for this.  I don't think we can mix a range (e.g. "50-63") with a single description can we?  I've never seen ranges used with "@Values" before.

Reply:
dont know... @peterbarker?

Reply:
The regex in the emitter won't match that. It would be better to type them out than to expand the regex of the emitter IMO.

Reply:
From a GCS consumer side I don't think there's any precedent or support for ranges like that. Values like this are usually used to form a selectbox, which doesn't work with a range very well.